### DISCUSSION TITLE
DIpy quickbundles issues numpy 2.0

### DISCUSSION THREAD

**Discussion by peterneher** [dipy/dipy#3284] [Tractography / Tractometry] [answered]
Hi everyone. I use dipy 1.9.0 together with numpy 2.0.0 and I am getting th following error

```
A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.0.0 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.
```

in

`dipy/segment/clustering.py", line 7, in <module>
    from dipy.segment.featurespeed import ResampleFeature
`

DO I need to downgrade numpy?

**Comment by skoudoro** [comment, marked as answer]
Yes, you need to downgrade Numpy.
Apologies for this.

The dev repository is compatible with Numpy 2.0 and we plan to release it this weekend or next weekend.

**Comment by peterneher** [comment]
No worries. I was just wondering why my nightly builds are failing, but if it's only a matter of weeks, I can wait. Thank you!
